compiler/dyno/lib/parsing/chpl.ypp

```diff
@@ -2262,11 +2263,11 @@ lambda_decl_expr:
       fp.formals = $2;
       fp.returnIntent = $3;
       fp.returnType = $4;
-      WhereAndLifetime wl = $5;
-      fp.body = $7;
-
+      fp.throws = ($5 == ThrowsTag_THROWS) ? true : false;
```

Comment:
why not `$5 == ThrowsTag_THROWS`?

Reply:
lolol it's true it's true, but in this case i just copy/pasted an existing line